### PR TITLE
chore: Fixes Linter

### PR DESCRIPTION
**Pull request by redreceipt**
### What does this PR do, or why is it needed?

Fixes the other PRs. Also provides a single source of truth for eslint packages on the core apps repo.

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by vinnyjth — APPROVED** [review]

**Comment by redreceipt** [comment]
may want to save your approval until I finish. looks like we were a tad behind and I need to go through and fix some actual lint errors.

**Comment by vinnyjth** [comment]
Go for it, then merge when ready. The direction you are taking looks good 👍

**Comment by redreceipt** [comment]
ok, so @vinnyjth the surface area on this one was bigger than I anticipated. It's a good thing, it's gonna clean up a lot of the eslint confusion we've seen over the project but I'd LOVE it if you'd take a quick glance over the changes before it's merged. if I couldn't quickly resolve, I ignored but I did attempt to fix some of it on some pretty important code.

File: packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ const ContentSingleFeatures = memo(
       <PaddedView vertical={false}>
         <H3 padded>{title}</H3>
       </PaddedView>
-      {features.map(({ __typename, ...feature }) => {
+      {[...features].map(({ __typename, ...feature }) => {
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js:20]: Fascinating. Do you remember what linter rule was being broken here?

Comment (redreceipt) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js:20]: Said map wasn't defined in prop types haha 🤷‍♂️

**Review by vinnyjth — APPROVED** [review]
This looks good to me. We'll probably need/want to adjust the templates project when this is merged, hopefully remove a lot of the devDeps.

File: packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js (hunk 1 above)
Comment (vinnyjth) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js:20]: lol!

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js:20]: Here's why, features is a `shape` instead of an `arrayOf` `shapes`. 

https://github.com/ApollosProject/apollos-apps/blob/96ac365821b0d28d56ebbdd9a415519820459f69/packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js#L38

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/ContentSingleFeaturesConnected/ContentSingleFeatures/index.js:20]: I'll fix this in propTypes, and remove the `blocked` label when I've done so.
